janus.c

```diff
@@ -36,38 +45,44 @@
 #include "auth.h"
 #include "utils.h"
 
+#include <openssl/err.h>
+#include <openssl/ssl.h>
 
 #define JANUS_NAME				"Janus WebRTC Gateway"
 #define JANUS_AUTHOR			"Meetecho s.r.l."
 #define JANUS_VERSION			12
 #define JANUS_VERSION_STRING	"0.1.2"
 #define JANUS_SERVER_NAME		"MyJanusInstance"
 
+#ifdef _WIN32
+#define SHLIB_EXT ".dll"
+#else
 #ifdef __MACH__
 #define SHLIB_EXT "0.dylib"
 #else
 #define SHLIB_EXT ".so"
 #endif
+#endif
 
 
-static janus_config *config = NULL;
-static char *config_file = NULL;
-static char *configs_folder = NULL;
+shared janus_config *config;
```

Comment:
Now this confuses me... why these static values are defined as shared instead? In this case, this would not result in them being static anymore on Linux.